tests/integration/_lib/fake-crawl-sites.js

```diff
@@ -0,0 +1,20 @@
+const { join } = require('path')
+const fs = require('fs')
+
+/** Folder under sandbox public that the sources scrape.
+ *
+ * These can be crawled with URLs like
+ * http://localhost:5555/tests/fake-source-urls/fake/fake.json. */
+const baseFolder = join(process.cwd(), 'public', 'tests', 'fake-source-urls')
+
+/** Write a test file to be scraped. */
+function writeFile (subdir, filename, content) {
```

Comment:
Why not just `module.exports = function writeFile`?

Reply:
I have a couple of ideas in mind for future things -- eg small test fixtures -- so we might want to do something like `function copyFixture(subdir, fixturePath)` later.